RefugeeLand.Core.Api/Controllers/RefugeesController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using RefugeeLand.Core.Api.Models.Refugees;
+using RefugeeLand.Core.Api.Services.Foundations.Refugees;
+using RESTFulSense.Controllers;
+
+namespace RefugeeLand.Core.Api.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class RefugeesController : RESTFulController
+{
+    private readonly IRefugeeService refugeeService;
+
+    public RefugeesController(IRefugeeService refugeeService)
+    {
+        this.refugeeService = refugeeService;
```

Comment:
Make it fat arrowed method

Reply:
Thx @ElbekDeveloper, I'll try to be more vigilant :)

Reply:
Done